### DISCUSSION TITLE
[Feature] Original scale display thumbnails

### DISCUSSION THREAD

**Discussion by linguowei** [immich-app/immich#1864] [Feature Request]
### The feature

Wish the thumbnails could be displayed in their original proportions instead of being cropped to a square

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by uhthomas** [comment]
Resolved by #2666?

**Reply by Oreo1980** [reply]
Only for the browser application, not for the mobile app. I would like to recognize the entire photo composition in the thumbnail gallery of the mobile app without opening up each and individual photo to do so.
